modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -1542,6 +1542,10 @@ cv::gimpl::ov::GOVExecutable::GOVExecutable(const ade::Graph &g,
                                             const std::vector<ade::NodeHandle> &nodes)
     : m_g(g), m_gm(m_g) {
 
+    if(cv::gapi::getCompileArg<std::reference_wrapper<cv::gapi::wip::ov::workload_type>>(compileArgs).has_value()) {
```

Comment:
Better to split this check for two lines: to retrieve compile arg and then check if it `has_value()`

Reply:
or might be create a brief alias for `std::reference_wrapper` here